antarest/launcher/adapters/local_launcher/local_launcher.py

```diff
@@ -89,59 +88,60 @@ def run_study(
                 self,
                 antares_solver_path,
                 study_uuid,
+                study_path,
                 job_id,
                 launcher_parameters,
             ),
             name=f"{self.__class__.__name__}-JobRunner",
         )
         job.start()
 
-    def _get_job_final_output_path(self, job_id: str) -> Path:
-        return self.config.storage.tmp_dir / f"antares_solver-{job_id}.log"
-
     def _compute(
         self,
         antares_solver_path: Path,
         study_uuid: str,
-        uuid: UUID,
+        study_path: Path,
+        job_id: str,
         launcher_parameters: LauncherParametersDTO,
     ) -> None:
         end = False
 
+        # create study logs
+        logs_path = study_path / "output" / "logs"
```

Comment:
As discussed:
to better separate responsibilities, like in the slurm launcher, we'd better not log straight to the study path, and remove it from the arguments.

We can have the same behaviour as in slurm launcher:
- while the computation is running or when it has failed, logs are in a directory managed by this service
- when computation succeeds, logs are copied to the actual output of the study